docs_user/modules/proc_migrating-ceph-mds.adoc

```diff
@@ -127,17 +97,17 @@ for item in $(ceph osd blocklist ls | awk '{print $0}'); do
 done
 ----
 
-== Migrate MDS to the target nodes
+[NOTE]
+When a file system client is unresponsive or misbehaving, it may happen that
+the access to the file system is forcibly terminated. This process is called
+eviction. Evicting a CephFS client prevents it from communicating further with MDS daemons and OSD daemons.
+Ordinarily, a blocklisted client may not reconnect to the servers: it must be unmounted and then remounted. However, in some situations it may be useful to permit a client that was evicted to attempt to reconnect. Because CephFS uses the RADOS OSD blocklist to control client eviction, CephFS clients can be permitted to reconnect by removing them from the blocklist.
 
-The MDS migration is performed by cephadm, and as done for the other daemons,
-the general idea is to move the daemons placement from a "hosts" based approach
-to a "label" based one. This ensures that the human operator can easily visualize
-the status of the cluster and where daemons are placed using the `ceph orch host`
-command, and have a general view of how the daemons are co-located within a
-given host, according to the https://access.redhat.com/articles/1548993[cardinality matrix]
-described in the associated article.
+.Procedure
 
-[source,bash]
+. The MDS migration is performed by cephadm, and as done for the other daemons, the general idea is to move the daemons placement from a "hosts" based approach to a "label" based one. This ensures that the human operator can easily visualize the status of the cluster and where daemons are placed using the `ceph orch host` command, and have a general view of how the daemons are co-located within a given host, according to the https://access.redhat.com/articles/1548993[cardinality matrix] described in the associated article. 
+//kgilliga: Is this a step? Can we rewrite it to something like: "Check the status of the Ceph cluster and export the MDS metadata"? Should this "cardinality matrix" article be available only downstream?
```

Comment:
It's not a step, it's a consideration that helps customers to better understand what they're going to do, which is the approach taken to migrate the Ceph daemons and the tools involved in the procedure. I think we can include this sentence in the `Note` above, and then provide an explicit `Step` for the procedure below.

```
. Get the host that are currently part of the Ceph cluster

---
[ceph: root@controller-0 /]# ceph orch host ls
...
...

. Extend the MDS labels to the target nodes:
..
..
```